examples/wschat.py

```diff
@@ -0,0 +1,37 @@
+from vyked import Host, WSService, ws, WebSocketResponse
+from aiohttp.web import MsgType
+
+
+class ChatWSService(WSService):
+
+    def __init__(self, ip, port):
+        super(ChatWSService, self).__init__("ChatService", "1", ip, port)
+        self.msgs = []
+
+    @ws(path='/')
+    def chat(self, request):
+        wsk = WebSocketResponse()
+        wsk.start(request)
+        while True:
```

Comment:
This looks awfully like the HTTPService but that is request-response based which doesn't have stream semantics like **wsk.receive()**. This should be more like TCPService where the TCP socket is not exposed.